### ISSUE TITLE
Dependency warnings on build

### ISSUE THREAD

**Issue by LaffertyDev** [codewinsdotcom/PostgresClientKit#38]
After installing a fresh `PostgresClientKit` using the Swift Package Manager, I received the following warning:

"Invalid exclude '/very/long/path/SourcePackages/checkouts/BlueSocket/Sources/....`: File not found."

There are a total of 8 warnings that have essentially the same message, for both `BlueSocket` and `BlueSSLService`.

I see a similar thing happening here, related to renaming the project... but I don't think that applies here. It looks like the old `IBM` repo was renamed to `Kitura`.

https://github.com/tid-kijyun/Kanna/pull/249/files

Any ideas?

**Comment by pitfield** [comment]
I see those compile warnings too.  These are coming from the BlueSocket and BlueSSLService dependencies, as you point out.  Despite these warnings, the build appears to succeed and work.

The problems in BlueSocket and BlueSSLService appear to have been fixed.  I changed PostgresClientKit to pick up a new release of the these dependencies, and the warnings no longer appear.  For now, I have committed this to the `develop` branch of PostgresClientKit (37e2010afbbf576cb9989456407010896c904d7b), if you'd like to try this.

Because this change uptakes new major versions of BlueSocket and BlueSSLService, I'm going to do some more testing before merging to master and releasing.

Thanks for reporting this.
